docs/processes/release/tasks/healthy-code.md

```diff
@@ -474,7 +474,7 @@ make check
 ## Test ICU4C Samples and Demos
 
 ### Windows build and test
-Note: Since ICU 73, this task has been included in the Azure DevOps Pipeline which is triggered automatically upon merging with main/maint* branches.
+Note: Since ICU 75, this task has been included in the GHA workflows which is triggered automatically upon merging with main/maint* branches.
```

Comment:
```suggestion
Note: Since ICU 76, this task has been included in the GHA workflows which is triggered automatically upon merging with main/maint* branches.
```